src/main/webapp/js/common.js

```diff
@@ -179,6 +181,19 @@ $(document).on('ajaxComplete ready', function() {
 });
 
 /**
+ * @param elements:
+ *     identifier that points to elements for which child 'img' tags are
+ *     checked for error and assigning default value in case of error.
```

Comment:
Something like this?
```java
/**
  * Binds a default image if the image is missing.
  * @param elements A collection of image elements. 
  */
```
The function name can be changed too. e.g. `bindDefaultImageIfMissing`
Furthermore, this function makes more sense if written for a single element and then the call site can call it in a loop.
The default image can be a parameter.